Gopkg.toml

```diff
@@ -1,14 +1,14 @@
 [[constraint]]
   name = "github.com/Comcast/codex"
-  version= "0.1.5"
+  version= "0.2.0"
```

Comment:
```suggestion
  version= "v0.2.0"
```

Reply:
modified and ran a new dep ensure update.